test/test_user.py

```diff
@@ -27,3 +30,24 @@ def test_can_create_new_user(self, helper):
         bobUserDetails = bob.get_user_details(helper.admin_user())
 
         print("Bob's user id is {}".format(bobUserDetails))
+
+    def test_create_duplicate_user(self, helper):
+        for num in range(2):
+            print("If a user DuplicateBob signs up")
+            try:
+                helper.given_new_fixed_user(
+                    "DuplicateBob"
+                )
+            except OSError as error:
+                print("DuplicateBob should not get created multiple times and returns a 422 error message")
+                error_string = str(error)
+                assert "request failed (422)" in error_string
+                print("There should be a JSON object in the error message that can be parsed")
+                json_object = error_string[(error_string.find(":")+1):]
```

Comment:
Something like `json_object = error_string.split(":", 1)[1:]`  would be a little cleaner

Reply:
Have made the change